common/src/main/java/tachyon/client/FileSystemMasterClient.java

```diff
@@ -350,6 +350,32 @@ public synchronized void setPinned(long fileId, boolean pinned) throws IOExcepti
   }
 
   /**
+   * Sets a new TTL value for the file. If the new TTL value equals {@link Constants#NO_TTL} and the
```

Comment:
Let's use full path to class `Constant`, so that we don't bring in unnecessary dependency at compilation.